src/pyskindose/plotting/create_setup_and_event_plot.py

```diff
@@ -1,27 +1,38 @@
 import logging
 from typing import List
+from .plot_settings import (
+    fetch_plot_colors,
+    fetch_plot_size,
+    fetch_plot_margin)
 
-import plotly.graph_objs as go
-import plotly.offline as ply
+import plotly.graph_objects as go
 
 from ..beam_class import Beam
 from ..constants import (
-    COLOR_AZURE_DARK,
     COLOR_BEAM,
     COLOR_DETECTOR,
     COLOR_PAD,
     COLOR_PATIENT,
-    COLOR_PLOT_TEXT,
     COLOR_SOURCE,
     COLOR_TABLE,
-    COLOR_ZERO_LINE,
     MESH_NAME_PAD,
     MESH_OPACITY_BEAM,
     PLOT_AXIS_TITLE_X,
     PLOT_AXIS_TITLE_Y,
     PLOT_AXIS_TITLE_Z,
     PLOT_FONT_FAMILY,
-    PLOT_HOVER_LABEL_FONT_FAMILY
+    PLOT_FONT_SIZE,
```

Comment:
Reorder to alphabetical order (for ease of reading later on

Reply:
resolved